external/troubleshooting/nvidia-troubleshooting.md

```diff
@@ -97,6 +97,32 @@ Tip: In case you only have an external monitor it is advised to attach it direct
 
 If you're seeing this error then that means another graphics card (most likely an integrated one) acted as failsafe. Disabling nouveau has the consequences of disabling nvidia support altogether.
 
+#### EFI
+
+ 1. Open a terminal in dom0.
+
+ 2. To edit the `xen.cfg` file using the `nano` editor:
+
+    ~~~ 
+    sudo nano /boot/efi/EFI/qubes/xen.cfg
+    ~~~
+
+ 3. Locate the three `kernel=` lines. Press the left/right arrow keys to position the cursor at the end of the lines, after `rhgb quiet`.
+
+ 4. Add the following:
+
+    ~~~
+    nouveau.modeset=0 rd.driver.blacklist=nouveau video=vesa:off
+    ~~~
+
+    This will temporarily disable nouveau until next boot.
```

Comment:
Editing this file won't be temporary. It will be effective until manually edited to the original state back.